app/jobs/create_onboarding_notifications_job.rb

```diff
@@ -32,6 +22,16 @@ def has_notification
   # only send once, so this is safe to run multiple times.
   SAFETY_OFFSET_IN_DAYS = 1
 
+  # 0 would be immediately after signing up
+  # 1 is a day after signing up, etc
+  #
+  # Each slug should be paired with a User::Notifications::Onboarding{$SLUG}Notification class
+  EMAILS = {
+    1 => :product,
+    3 => :community,
+    5 => :fundraising
+  }.map { |day, slug| OnboardingEmail.new(day:, slug:) }.freeze
+
```

Comment:
Does this actually change something or just reorder?

Reply:
Reverted.